README.md

```diff
@@ -60,10 +60,6 @@ cd <path/to/SMARTS>
 # Install the system requirements, then follow the instructions for setting up the SUMO_HOME environment variable. You may use the `-y` option to enable automatic assumption of "yes" to all prompts to avoid timeout from waiting for user input. 
```

Comment:
I think this comment needs to be updated too.

Reply:
yes, and after we eventually move sumo to optional, we might need to modify this README.md again.